scripts/print-bottles-deps.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+import sys
+
+binaries = [
+    "tezos-client",
```

Comment:
v18.0-rc1 introduced new `*-dac` binaries with the corresponding formulae. Could you please check them as well?

Reply:
Sure, I will as soon as v18.0 is published

Reply:
`v18.0-rc1` and corresponding formulae are already published, btw

Reply:
Seems that `dac` files also have `libffi` dependency which they actually do not use. I will remove the dependency now. Do you want to check anything else here, @rvem